tools/data-handler/src/create.ts

```diff
@@ -155,15 +154,15 @@ export class Create extends EventEmitter {
    * @param {string} templateName Template name to add cards into.
    * @param {string} card Optional, if defined adds a new child-card under the card.
    * @param {number} count How many cards to add. By default one.
-   * @returns string with information about the operation
+   * @returns non-empty string array with ids of added cards
```

Comment:
ids -> card keys; the block could have @throws information as well (not related to this ticket)